pkg/resourcemanager/mysql/mysqluser/mysqluser_reconcile.go

```diff
@@ -23,6 +23,8 @@ import (
 	keyvaultSecrets "github.com/Azure/azure-service-operator/pkg/secrets/keyvault"
 )
 
+const mysqlDatabase = "mysql"
```

Comment:
Same comment about possibly worth leaving a "what the heck is this" comment? I think this is a magical DB name that is the default DB but not 100% sure